programmers/두개-뽑아서-더하기.js

```diff
@@ -0,0 +1,13 @@
+function solution(numbers) {
+    const result = [];
+    
+    for (let i = 0; i < numbers.length - 1; i++) {
+        for (let j = i + 1; j < numbers.length; j++) {
+            result.push(numbers[i] + numbers[j]);
+        };
+    };
+    
+    const set = new Set(result);
+    
+    return [...set].sort((a, b) => a - b);
```

Comment:
```suggestion
   rreturn [...new Set(answer.sort((a, b) => a - b))];
```
변수 할당은 보기에 깔끔해서 좋지만 메모리를 사용해야 하므로  선언 후 여러 번 사용하는 경우가 아니라면 변수 선언을 지양하는 것이 좋습니다. 

전개 연산자를 사용하신 것은 정말 대단하신 것 같아요. 👍